### PR TITLE
feat: GEOT-784: header changes

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/app-andrew/src/pages/group/styles.scss [hunk 1]
```diff
@@ -69,12 +61,78 @@
...
+
+    &.overlay {
+      &:before {
+        content: '';
+      }
+    }
+
+    > div {
+      position: relative;
+      z-index: 11 !important;
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/group/styles.scss:88]: I'll clean up z-indexes in next PR

File: packages/app-andrew/src/components/group/GroupHeader/styles.scss [hunk 2]
```diff
@@ -1,14 +1,21 @@
 @import '~@gtms/styles/scss/variables.scss';
 @import '~@gtms/styles/scss/mixins/imageCover.scss';
 
-.groupHeader {
+.groupHeaderComponent {
+  position: relative;
+  z-index: 10;
   display: flex;
   gap: 8px;
```
Comment (ra8ga) [inline packages/app-andrew/src/components/group/GroupHeader/styles.scss:8]: in next PR I will create variable for all 8px

File: packages/app-andrew/src/components/group/GroupHeader/styles.scss [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+  width: 100%;
   max-width: $tempWidth;
+  padding: 8px 0px;
```
Comment (ra8ga) [inline packages/app-andrew/src/components/group/GroupHeader/styles.scss:11]: I will create variable for all 8pxs

File: packages/app-andrew/src/pages/group/styles.scss (hunk 1 above)
Comment (mariusz-kabala) [reply packages/app-andrew/src/pages/group/styles.scss:88]: here we go again...

File: packages/app-andrew/src/components/group/GroupHeader/styles.scss [hunk 4]
```diff
@@ -1,14 +1,21 @@
...
+  transition: top 200ms linear;
 
   .templogo {
-    min-width: 40px;
-    min-height: 40px;
+    min-width: 35px;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group/GroupHeader/styles.scss:15]: just with, wtf is that

Comment (ra8ga) [reply packages/app-andrew/src/components/group/GroupHeader/styles.scss:15]: this needs to be removed anyway - left it as it is bcause it works

**Review by mariusz-kabala — APPROVED** [review]

**Pull request merged**